client/src/components/menubar/style.ts

```diff
@@ -0,0 +1,75 @@
+import styled from "@emotion/styled";
+import { spacesS } from "../theme";
+import { getFeatureFlag } from "../../util/featureFlags/featureFlags";
+import { FEATURES } from "../../util/featureFlags/features";
+
+export const fullVerticalScreenWidth = 500;
+const isTest = getFeatureFlag(FEATURES.TEST);
+const isDownload = getFeatureFlag(FEATURES.DOWNLOAD);
+
+let firstBreakpoint = "655px";
+if (isTest && isDownload) {
+  firstBreakpoint = "705px";
+} else if (isTest || isDownload) {
+  firstBreakpoint = "685px";
+}
+
+export const MenuBarWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 100%;
+  position: relative;
+  container: menu-bar / inline-size;
+`;
+
+export const ResponsiveMenuGroupOne = styled.div`
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: wrap;
+  justify-content: right;
+  @container menu-bar (max-width: ${fullVerticalScreenWidth}px) {
+    flex-direction: column-reverse;
+    position: absolute;
+    top: 37px;
+  }
+`;
+
+export const ResponsiveMenuGroupTwo = styled.div`
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: wrap;
+  justify-content: right;
+  @container menu-bar (max-width: ${firstBreakpoint}) {
```

Comment:
Hmm my VSCode is not showing error for `@container` 🤔 Can you take a screenshot of the error you see please 😄 ?

Thank you!

Reply:
This is what it looks like on mine - I think it is just an out of date extension.
![Screenshot 2024-07-29 at 2 16 20 PM](https://github.com/user-attachments/assets/b6f46d10-6d50-4de7-9aad-d4be660e5b29)